packages/formation/sass/base/_b-variables.scss

```diff
@@ -31,34 +31,39 @@ $h6-font-size:        1.5rem;
 $text-max-width:      70rem; // Note: USWDS value is 53rem;
 $site-max-width:      100rem; // Worksout to about 1000px. USWDS value is 1040px
 
+
+//================
+//  USWDS Colors
```

Comment:
Formation uses USWDS v1 so a minor nit here would be that we mention this is USWDS v3 for clarity.